src/components/styledComponents.js

```diff
@@ -37,7 +38,7 @@ const Text = styled( UnstyledText );
 // $FlowIgnore
 const TextInput = styled( UntyledTextInput );
 // $FlowIgnore
-const Pressable = styled( StyledPressable );
+const Pressable = styled( PressableWithTracking );
```

Comment:
Should we import as UnstyledPressableWithTracking for consistency?

Reply:
yep, good call